src/System.Runtime.Serialization.Formatters/tests/EqualityExtensions.cs

```diff
@@ -100,12 +100,22 @@ public static bool IsEqual(this DataTable @this, DataTable other)
 
         public static bool IsEqual(this Comparer @this, Comparer other)
```

Comment:
All the equality checks should also be refined in future to test the whole object. It's on my list.